documentation/tags.md

```diff
@@ -209,6 +212,7 @@ Readr
 Recursion
 Redux
 References
+Referential Transparency
```

Comment:
```suggestion
References
```

Like "Pure Functions", "Referential Transparency" is a little too specific even for a tag.